client/src/components/Dashboard/Dashboard.js

```diff
@@ -19,7 +19,11 @@ export default function Dashboard(props) {
 
   async function fetchBoardList() {
     try {
-      const result = await axios(SERVER_URL + "/dashboard/getboards");
+      const result = await axios("/dashboard/getboards",{
```

Comment:
I think you forgot to append SERVER_URL with the URL "/dashboard/getboards"

Reply:
We are using proxy to hit the backend request to get cookies in the backend. If we get time, later on, we will switch to server URL